packages/embed-react/README.md

```diff
@@ -158,10 +158,30 @@ Embed will automatically submit the payment form with hidden inputs, this can be
 />
 ```
 
+### Submission without a Form
+
+Embed does not require a form to be present, in order to access the `EmbedInstance` directly you can pass a ref.
+
+```tsx
+import Gr4vyEmbed, { EmbedInstance } from '@gr4vy/embed-react';
+import { useRef } from 'react';
+
+const MyComponent = () => {
+  const embed = useRef<EmbedInstance>();
+
+  return <form>
+    <Gr4vyEmbed ref={embed} {...}/>
+    <button onClick={() => embed.current.submit()}>
+  </form>
+
+}
```

Comment:
This example is a tad confusing. You say a `form` is not required but then you use a `form` in the example. Maybe replace it with a `div`?

Reply:
It's still good practice to use a form for semantics (a11y) - its just we won't hijack the form and take over the submit.
Happy to change to a div though.